src/JustEat.StatsD.Tests/WhenUsingPooledUdpTransport.cs

```diff
@@ -42,7 +42,7 @@ public static void MultipleMetricsCanBeSentWithoutAnExceptionBeingThrownParallel
 
             using (var target = new PooledUdpTransport(endPointSource))
             {
-                Parallel.For(0, 10_000, (_) =>
+                Parallel.For(0, 10_000, _ =>
```

Comment:
I though you and Martin liked to always include braces. Is this a change of heart?

Reply:
I put the braces there, then Anthony took them away again 🤣

Reply:
I'm not a fan of unnecessary braces.